.travis.yml

```diff
@@ -1,8 +1,15 @@
 os: linux
-dist: xenial
+dist: bionic
 language: minimal
 services:
 - docker
 script:
-- ./build.sh _travis-${TRAVIS_OS_NAME}
-- ./scripts/travis_test.sh
+- ./build.sh linux
+- find . -name ooniprobe -type f
+- ./debian/build_package.sh
```

Comment:
I think this should only be invoked when on a tag, unless we're talking about nightly packages here

Reply:
I guess it doesn't hurt to check if we can build the package on every PR, no?